src/create.rs

```diff
@@ -929,7 +875,7 @@ impl Execution {
         // for PathBuf, but it was unexpected and kind of annoying because I am
         // not sure how to add a trailing slash in a cross-platform way with
         // PathBuf, not that cargo-wix needs to be cross-platform.
-        dst.push(format!("{}\\", WIX));
+        let dst = target_directory.join("");

```

Comment:
So, I had the same problem to add a trailing backslash... I essentially found two ways, this one, or

```rust
let s = target_directory.as_os_string();
s.push("\\");
PathBuf::from(s)
```